agentforce/1.2/modules/ROOT/pages/configuring-agent-operations.adoc

```diff
@@ -0,0 +1,70 @@
+= Configuring Agent Operations for Agentforce Connector 1.2 
+
```

Comment:
Add an intro sentence to this page. For example:

Use these guidelines to configure the *Start agent conversation*, *Continue agent conversation*, and *End agent conversation* operations for Agentforce Connector.